bazel/src/main/java/com/google/api/codegen/bazel/ApiVersionedDir.java

```diff
@@ -361,9 +366,11 @@ void parseBazelBuildFile(Path file) {
             // Duplicated rule of the same kind will break our logic for preserving rule name.
             System.err.println("There are more than one rule of kind " + kind + ".");
             System.err.println(
-                "Bazel build file generator does not support regenerating BUILD.bazel in this case.");
+                "Bazel build file generator does not support regenerating BUILD.bazel in this"
```

Comment:
Since we are already touching this code and split it into concatenated strings, can you please move all that into single System.err.println() (instead of using subseuent calls to print essentially a single multi-line error message)?

Reply:
Done.